### ISSUE TITLE
use-symbolic-message-instead should be case-insensitive for numerical IDs

### ISSUE THREAD

**Issue by I-g-e** [PyCQA/pylint#5000]
### Bug description

When `use-symbolic-message-instead` is enabled, Pylint will correctly warn about the usage of non-symbolic ID in pragmas, like here.

```python
# pylint: disable=W0223
```

This will output `I0023: 'W0223' is cryptic: use '# pylint: disable=abstract-method' instead (use-symbolic-message-instead)`.

But if the category letter of the ID is written *in lower case*, like here:

```python
# pylint: disable=w0223
```
Pylint will **not** output the warning about not using a symbolic message.

### Configuration

```ini
[MASTER]
enable=use-symbolic-message-instead
```

### Command used

```shell
pylint test.py
```

### Pylint output

No output.

### Expected behavior

Since Pylint does allow lower case in the pragmas that use a numeric ID (e.g. `w0223`), having `use-symbolic-message-instead` enabled should also output a message regardless of the case of the ID used in the `disable` pragma.

### Pylint version

```shell
pylint 2.10.2
astroid 2.7.3
Python 3.8.10 (default, May  4 2021, 00:00:00) 
[GCC 10.2.1 20201125 (Red Hat 10.2.1-9)]
```

**Comment by Pierre-Sassoulas** [comment]
Thank you for opening the issue, I can reproduce that. Strangely ``c0111`` works but not ``w0223``. Might be harder to troubleshoot than I thought.

**Comment by Pierre-Sassoulas** [comment]
It was due to C0111 being correctly capitalized once in the functional test I was using. It should be fixed in 2.11 :)
